gitpod-io/gitpod
# fix(supervisor): typo in ResourcesStatusRequest

## Disclaimer

I am happy to drop the PR if the review is too complicated / long.

## Description
Small PR just fixing a typo in our code `ResourcesStatuRequest` -> `ResourcesStatusRequest`

## Related Issue(s)

N/A

## How to test
1. Start a workspace in this prev env
2. Make sure `gp top` still works

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:
